.github/workflows/misc-dailies.yml

```diff
@@ -42,99 +42,29 @@ jobs:
   coverity-auth:
     name: coverity scan of the auth
     if: ${{ vars.SCHEDULED_MISC_DAILIES }}
-    runs-on: ubuntu-22.04
-    env:
-      COVERITY_TOKEN: ${{ secrets.coverity_auth_token }}
-      FUZZING_TARGETS: no
-      SANITIZERS:
-      UNIT_TESTS: no
-    steps:
-      - uses: PowerDNS/pdns/set-ubuntu-mirror@meta
-      - uses: actions/checkout@v4
-        with:
-          fetch-depth: 5
-          submodules: recursive
-      - uses: actions/setup-python@v5
-        with:
-          python-version: '3.11'
-      - run: build-scripts/gh-actions-setup-inv-no-dist-upgrade
-      - run: inv install-clang
-      - run: inv install-auth-build-deps
-      - run: inv install-coverity-tools PowerDNS
-      - run: inv coverity-clang-configure
-      - run: inv ci-autoconf
-      - run: inv ci-auth-configure
-      - run: inv coverity-make
-      - run: inv coverity-tarball auth.tar.bz2
-      - run: inv coverity-upload ${{ secrets.coverity_email }} PowerDNS auth.tar.bz2
+    uses: PowerDNS/pdns/.github/workflows/coverity.yml@master
```

Comment:
LGTM!

The only thing I am missing is that this change will require adding the permission `actions: read` for this workflow

Reply:
Thanks, it's always very hard for me to get that right without trial and error.. Do you have a good resource on the subject, by any chance?
I pushed a commit that hopefully should fix it, but please check it carefully :)

Reply:
Well... it turns out I was wrong. I did need this tag in the past but it seems it is not required anymore (I tested it)

Also, the documentation says it is only required for private repositories [https://docs.github.com/en/rest/actions/workflows?apiVersion=2022-11-28#get-a-workflow](https://docs.github.com/en/rest/actions/workflows?apiVersion=2022-11-28#get-a-workflow)

From what I tested, it is safe to remove that permission

Reply:
Ah, I tried hard to find that information but did not manage to, well done! :) I'll remove the last commit then, worst case scenario Coverity scans will be broken for a day.

Reply:
Removed!